### PR TITLE
:sparkles: Update stakeholder groups table to use ActionsColumn

### PR DESCRIPTION
**Pull request by MiriSafra**
Addition ActionsColumn to  Stakeholder groups table
Relates to https://github.com/konveyor/tackle2-ui/issues/1318
UI tests PR: 1222

### CODE REVIEW

File: client/src/app/pages/controls/stakeholder-groups/stakeholder-groups.tsx [hunk 1]
```diff
@@ -284,7 +284,14 @@ export const StakeholderGroups: React.FC = () => {
...
                             {stakeholderGroup.stakeholders?.length}
                           </Td>
-                          <AppTableActionButtons
+                          {/* <AppTableActionButtons
+                            onEdit={() =>
+                              setCreateUpdateModalState(stakeholderGroup)
+                            }
+                            onDelete={() => deleteRow(stakeholderGroup)}
+                          />
+                        </TableRowContentWithControls> */}
```
Comment (mguetta1) [inline client/src/app/pages/controls/stakeholder-groups/stakeholder-groups.tsx:293]: @MiriSafra please remove comments

File: package-lock.json
Comment (mguetta1) [inline package-lock.json:1]: No need to modify `package-lock.json` file

**Review by mguetta1 — CHANGES_REQUESTED** [review]
Thanks @MiriSafra for the PR. Please see my comments

**Review by mguetta1 — APPROVED** [review]
LGTM

**Comment by mguetta1** [comment]
Closed and reopened the PR to update the description with UI tests PR

**Review by rszwajko — APPROVED** [review]

**Comment by rszwajko** [comment]
@MiriSafra PR looks good - I've changed the title so that it would look more like previous work on this topic - see https://github.com/konveyor/tackle2-ui/pull/1923

**Pull request merged**